compiler-rs/openapi_to_clients_schema/src/main.rs

```diff
@@ -26,31 +26,40 @@ fn main() -> anyhow::Result<()> {
     let subscriber = FmtSubscriber::builder().with_max_level(Level::TRACE).finish();
     tracing::subscriber::set_global_default(subscriber)?;
 
-    // let file = "../../ent-search/swagger/v1/enterprise-search.json";
-    // let file = "../../ent-search/swagger/v1/workplace-search.json";
-    let file = "./openapi_to_clients_schema/fixtures/workplace-search.json";
+    let path = "../../../ent-search/swagger/enterprise-search.json";
```

Comment:
The file in fixtures/ was empty, so I deleted it and pointed to a more likely way to find it instead.

Reply:
What about adding an `env` var and keeping this as default?

Reply:
I was considering addressing that as a follow-up, using `std::env::args` and then use `cargo run -- fixtures/kibana.serverless.yaml`. As described in https://doc.rust-lang.org/book/ch12-01-accepting-command-line-arguments.html. It feels better than an env var as my shell will provide autocompletion. (And clap seems overkill here.)

Reply:
OK, I ended up doing it in [`8cf465a` (#3195)](https://github.com/elastic/elasticsearch-specification/pull/3195/commits/8cf465a83449971b079b260da3968b3b5583439a). Please tell me what you think.

(I also tried using `Err(anyhow::anyhow!("expected a single argument"))` instead of panicking. It gave a better error, but it also leads to uglier code as I had to use `String` instead of `&str`.)